### PR TITLE
Add post routes method to allow posting of routes to database

### PR DESCRIPTION
**Pull request by sahilpatel1906**

### CODE REVIEW

**Comment by Aebel-Shajan** [comment]
This is similar to what we did in #70. But #70 has additional modifications to the backend. We should review both before merging to main.

**Review by Aebel-Shajan — COMMENTED** [review]
We already did this in #70

**Review by Jean-005 — APPROVED** [review]
Very Nice!

**Review by ney601 — APPROVED** [review]

**Review by Tommybui28 — APPROVED** [review]

**Pull request closed without merging**
